src/main/java/uk/gov/hmcts/reform/preapi/entities/Case.java

```diff
@@ -48,6 +51,13 @@ public class Case extends CreatedModifiedAtEntity implements ISoftDeletable {
     @Transient
     private boolean isSoftDeleteOperation;
 
+    @Column(name = "state", nullable = false)
```

Comment:
This will need to be nullable initially so that we can release the code without breaking backward compatability.
Please raise another jira ticket to update that to set nullable to false again once we have release the app portion of this feature.